vivisect/parsers/macho.py

```diff
@@ -37,7 +37,7 @@ def _loadMacho(vw, filebytes, filename=None, baseaddr=None):
     sha256 = viv_parsers.sha256Bytes(filebytes)
 
     # Check for the FAT binary magic...
-    if binascii.hexlify(filebytes[:4]) in (b'cafebabe', b'bebafeca'):
+    if e_common.hexify(filebytes[:4]) in ('cafebabe', 'bebafeca'):
```

Comment:
huh?  why convert to compare the strings instead of bytes?  this one confuses me on principle.  i'm not going to hold up the merge on this one.  just curious.

Reply:
Honestly? This was just for conformity.